bee-crypto/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Deprecated
 
+- Deprecated `CurlP` from `bee-crypto` (now uses `iota-crypto` instead)
+
```

Comment:
Can you actually prepare a release ? Maybe `0.3.0`, it's probably time to drop the alpha tag anyway.